wurst/file/SaveLoadData.wurst

```diff
@@ -46,7 +46,7 @@ public interface LoadListener
 public function player.saveData(string slotName, string data)
 	let buffer = new ChunkedString(DEFAULT_CHUNK_SIZE)
 	buffer.append(data)
-	this.saveData(slotName.endsWith(".pld") ? slotName : slotName + ".pld", buffer)
```

Comment:
Last time i checked, `.pld` ending was required for the fileIO to work.
Have you tested this?

Reply:
It is required, but it’s handled by the other overload of the function - the one taking a ChunkedString. The edited overloaded basically just converts a string into a ChunkedString and the file ending was handled twice.

Reply:
It is required, but it’s handled by the other overload of the function - the one taking a ChunkedString. The edited overloaded basically just converts a string into a ChunkedString and the file ending was handled twice.